pages/draw.md

```diff
@@ -262,6 +262,37 @@ poi-icons:
             sprite_default: generic
 ```
 
+####`collide`
+Optional _boolean_. Defaults to `true`.
+
+Applied to `points` and `text`
+
+```yaml
+poi-icons:
+    draw:
+        points:
+           collide: false
+```
+
+####`transition`
+Optional _map_ , where key is one or both of `hide` and `show` and value is a _map_ of `time` to time.
+Time values can be either in seconds (`s`) or milliseconds (`ms`).
+
+Sets the transition time from hide to show for points or texts!
+
+A transition time of `0` means, the point or text is persistent.
```

Comment:
Saying that the point or text is persistent with a transition of `0` is a bit confusing. A transition of `0` means instantaneous transition to non-visible state.